### ISSUE TITLE
Is DELETE supported

### ISSUE THREAD

**Issue by philbarker** [ox-it/ords-ui#98]
Is it possible to write a query to delete rows?

**Comment by scottbw** [comment]
No. You can delete individual records using the browse interface (which makes an REST API call), or you can execute any SQL you like via ODBC (depending on role), but ad-hoc queries through the browser use a read-only connection.
